nomad/structs/structs.go

```diff
@@ -859,6 +859,8 @@ type EvalDequeueRequest struct {
 type EvalListRequest struct {
 	FilterJobID      string
 	FilterEvalStatus string
+	FilterNamespace  string
```

Comment:
Do we need a new namespace filter? What's the difference from `QueryOptions.Namespace`?